### PR TITLE
Rates style and form update

### PR DESCRIPTION
**Pull request by anoopgcmz**

### CODE REVIEW

File: app/scripts/controllers/rates-dialog-ctrl.js [hunk 1]
```diff
@@ -8,81 +8,80 @@
...
+                $scope.datas.hours.push(value);
+            }
+
+            $scope.datas.hourslength = $scope.datas.hours.length;
+        };
+
+        $scope.submitForm = function () {
+            var data = angular.copy($scope.datas);
+
+            //data.start_date = '2000-01-01';
```
Comment (gchablowski) [inline app/scripts/controllers/rates-dialog-ctrl.js]: comment out

File: app/styles/app.css [hunk 2]
```diff
@@ -2366,6 +2366,10 @@ md-select.course {
     height: 140px;
 }
 
+.card-spacing{
+    margin-right:15px; 
+    padding-bottom:20px !important;
```
Comment (gchablowski) [inline app/styles/app.css]: put out the !important. That bad use in general.

File: app/scripts/controllers/rates-dialog-ctrl.js (hunk 1 above)
Comment (anoopgcmz) [reply app/scripts/controllers/rates-dialog-ctrl.js]: removed and updated

File: app/styles/app.css (hunk 2 above)
Comment (anoopgcmz) [reply app/styles/app.css]: updated by adding height and removed padding.

**Pull request merged**